src/CheckoutSdk/Payments/ProductType.cs

```diff
@@ -4,9 +4,40 @@ namespace Checkout.Payments
 {
     public enum ProductType
     {
-        [EnumMember(Value = "QR Code")] QrCode,
-        [EnumMember(Value = "In-App")] InApp,
-        [EnumMember(Value = "Official Account")]OfficialAccount,
-        [EnumMember(Value = "Mini Program")] MiniProgram
+        [EnumMember(Value = "QR Code")]
+        QrCode,
+
+        [EnumMember(Value = "In-App")]
+        InApp,
+
+        [EnumMember(Value = "Official Account")]
+        OfficialAccount,
+
+        [EnumMember(Value = "Mini Program")]
+        MiniProgram,
+
+        [EnumMember(Value = "pay_in_full")]
+        PayInFull,
+
+        [EnumMember(Value = "pay_by_instalment")]
```

Comment:
Why are the new enum vals a different format than the older ones?

Reply:
To read easier: https://api-reference.checkout.com/#operation/requestAPaymentOrPayout!path=0/processing/product_type&t=request